projects/packages/analyzer/src/class-utils.php

```diff
@@ -31,24 +31,37 @@ static function get_param_default_as_string( $default, $current_class ) {
 		}
 	}
 
-	static function node_to_class_name( $node, $class_for_self = null ) {
+	/**
+	 * Gets string representation of the Node's class
+	 *
+	 * @param Node $node Node.
+	 * @param any  $class_for_self Class reference.
+	 * @return string
+	 */
+	public static function node_to_class_name( $node, $class_for_self = null ) {
 		if ( $node instanceof Node\Expr\Variable
 			|| $node instanceof Node\Stmt\Class_ ) {
 			$class_name = $node->name;
```

Comment:
When it's `Node\Expr\Variable`, should the result actually be `'$' . self::maybe_stringify( $node->name )` like I see is being done elsewhere?